logging.adoc

```diff
@@ -0,0 +1,384 @@
+. `otlp` `url` specify a valid host:port of the otel receiver and *MUST* terminate with "*/v1/logs*"
+. `otlp` `tuning` `compression` default value is "*gzip*". Specify "*none*" to turn compression off.
+. `otlp` `authentication` is optional and specifies a `token` `from` and a value of "*serviceAccount*"
+.. The token can also be read from a secret
+.. Also available with `username` and `password` authentication spec (refer to HTTP Auth Specification for full scope)
+
+
+.TLS InsecureSkipVerify
+[NOTE]
+This option is *NOT* recommended for production configurations. If true, the client will be configured to skip validating server certificates.
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/clusterlogforwarder_types.go#L244[TLS Specification] for _observability.openshift.io/v1_
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L203[HTTP Auth Specification] for _observability.openshift.io/v1_
+
+https://github.com/openshift/cluster-logging-operator/blob/master/api/observability/v1/output_types.go#L812[Tuning Specification] for _observability.openshift.io/v1_
+
+
+== Data Model
+
+=== Semantic Convention
+The Semantic Conventions in OpenTelemetry define a *Resource* as an immutable representation of the entity producing telemetry as *Attributes*.
+
+For example, a process producing telemetry that is running in a container has a container_name, a cluster_id, a pod_name, a namespace, and possibly a deployment or app_name.
+
+All of these *Attributes* are included in the *Resource* object.  This grouping and reducing of common attributes is a powerful tool when sending logs as telemetry data.
+
+.OTEL References
+* https://opentelemetry.io/docs/specs/semconv/[Semantic Conventions]
+* https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+* https://opentelemetry.io/docs/specs/semconv/general/logs/[General Logs Attributes]
+
+=== Data Mapping
+This document is not a complete spec and is mainly used as a guide.
+
+==== JSON Encoding
+OpenShift Logging uses Vector Remap Language https://vector.dev/docs/reference/vrl/[(VRL)] to map and enrich the logging data in order to the follow the semantic conventions of OpenTelemetry and its https://opentelemetry.io/docs/specs/otel/logs/data-model/[Logs Data Model]
+
+https://github.com/open-telemetry/opentelemetry-proto/blob/v1.3.1/examples/logs.json[Example JSON Encoded Logs] provide a visual representation of the data structure
+
+[%header,format=csv]
+|===
+ViaQ,OTEL
+openshift.cluster_id,       resource.attributes.cluster.id
+hostname,	                resource.attributes.node.name
+kubernetes.namespace_name,	resource.attributes.k8s.namespace.name
+kubernetes.pod_name,	    resource.attributes.k8s.pod.name
+log_source,	                resource.attributes.openshift.log.source
+kubernetes.container_name,	resource.attributes.k8s.container.name
```

Comment:
I ask because I do not know the answer.  Do we need to identify "resource" or "logrecord" attributes?  From OTEL perspective don't this just all get merged together?  If I query for some specific attribute do I need to know its a resource, span or logRecord?  If the answer is it does not matter then I would drop these prefixes